sqladmin/templates/_macros.html

```diff
@@ -1,32 +1,33 @@
 {% macro menu_category(menu, request) %}
-{% if menu.is_visible(request) and menu.is_accessible(request) %}
-<li class="nav-item dropdown">
-  <a class="nav-link dropdown-toggle {% if menu.is_active(request) %}active{% endif %}" data-bs-toggle="dropdown"
-    href="#">
-    <span class="nav-link-icon d-md-none d-lg-inline-block">
-      {% if menu.icon %}<i class="{{ menu.icon }}"></i>{% endif %}
-    </span>
-    <span class="nav-link-title">{{ menu.display_name }}</span>
-  </a>
-  <div class="dropdown-menu show">
-    <div class="dropdown-menu-columns">
-      <div class="dropdown-menu-column">
-        {% for sub_menu in menu.children %}
-        <a class="nav-link ps-lg-3 {% if sub_menu.is_active(request) %}active{% endif %}"
-          href="{{ sub_menu.url(request) }}">
-          <span class="nav-link-icon d-md-none d-lg-inline-block">
-            {% if sub_menu.icon %}<i class="{{ sub_menu.icon }}"></i>{% endif %}
-          </span>
-          <span class="nav-link-title">{{ sub_menu.display_name }}</span>
-        </a>
-        {% endfor %}
+  {% if  menu.is_active(request) %}
```

Comment:
Why are you switching the is_visible and is_accessible to is_active only here?

Reply:
Hi, I switch is_visible and is_accessible to is_active only here, because we are checking here CategoryMenu and elsewhere ItemMenu. And ItemMenu has is_active = False

![image](https://github.com/aminalaee/sqladmin/assets/82371930/3c00add4-4475-4360-8c90-139b3e83d5af)

Reply:
I've tested everything, everything works great!